src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -159,6 +159,15 @@ private static ArgumentParser getArgumentParser() {
                         + "If not provided, default file formats will be used.\n"
                         + "Please refer to userguide for more information.");
 
+        parser.addArgument(TIMEZONE_FLAGS)
```

Comment:
Any reason for shifting it here? Or just arrangement?

Reply:
This was originally at the bottom after the mutex flags. When I added in the 2 new mutex flags, I moved this above all the mutex flags so it would be a bit clearer.

Reply:
Yes, it will be clearer now.